src/OrchardCore.Modules/OrchardCore.ContentFields/GraphQL/Startup.cs

```diff
@@ -22,4 +23,13 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddObjectGraphType<UserPickerField, UserPickerFieldQueryObjectType>();
         }
     }
+
+    [RequireFeatures("OrchardCore.Apis.GraphQL", "OrchardCore.ContentFields.Indexing.SQL")]
+    public class IndexStartup : StartupBase
```

Comment:
Please document this new feature under https://docs.orchardcore.net/en/latest/reference/modules/SQLIndexing/#content-fields-indexing. Otherwise, it won't be apparent that SQL Index has GraphQL support.

Reply:
Done